### ISSUE TITLE
Introduce local from "else if" generates local inside else.

### ISSUE THREAD

**Issue by Pilchie** [dotnet/roslyn#561]
``` C#
class C
{
    static void Main(string[] args)
    {
        if (true)
        {
        }
        else if (args.Length == 0)
        {
        }
    }
}
```

Select `args.Length == 0` and use Ctrl+. to invoke introduce local.

**Result**

``` C#
        if (true)
        {
        }
        else
        {
            var v = args.Length == 0;
            if (v)
            {
            }
        }
```

**Expected**
Would be nicer to put the local before the if itself - while it changes semantics if there are side-effects, it's more likely what someone would want.

**Note**
Was reported through the Send-a-smile/Send-a-frown.
